test/cfg/std.cpp

```diff
@@ -4154,7 +4154,7 @@ void uninitvar_setbase(void)
     std::cout << std::setbase(p);
 }
 
-// cppcheck-suppress passedByValue
+// TODO cppcheck-suppress passedByValue
```

Comment:
Do you plan to address these in this PR? If not, please open a ticket about it as it is a regression. 

We probably should have done this in several other cases in the past...

Reply:
I think the issue is https://trac.cppcheck.net/ticket/12087
`isVariableChanged()` uses the container action somewhere to decide constness.